components/gc-featured-link/gc-featured-link-fr.html

```diff
@@ -1,116 +1,96 @@
 ---
-breadcrumb: [
-	{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/accueil.html" }
-]
-dateModified: "2021-06-03"
-description: "Documentation et exemple pratique de lien en vedette - fonctionnalité provisoire"
-language: "fr"
-altLangPage: "gc-featured-link-en.html"
-title: "Lien en vedette - fonctionnalité provisoire"
+title: Lien en vedette - fonctionnalité provisoire
+description: Documentation et exemple pratique de lien en vedette - fonctionnalité provisoire
+language: fr
+altLangPage: gc-featured-link-en.html
+dateModified: 2021-07-02
+layout: no-container
 ---
-{% include alert-provisional.html siteroot="../../" %}
+<div class="container">
+	<h1>{{ page.title }}</h1>
 
-<span class="wb-prettify all-pre hide"></span>
+	{% include alert-provisional.html siteroot="../../" %}
 
-<dl class="horizontal">
-	<dt>Statut</dt>
-	<dd>Provisiore</dd>
-	<dt>Type</dt>
-	<dd>Conception de Canada.ca</dd>
-	<dt>Dernière revue</dt>
-	<dd>2021-06-02</dd>
-	<dt>Est conforme à</dt>
-	<dd>Spécifications du contenu et de l'architecture de l'information 2.1.2</dd>
-	<dt>Directives</dt>
-	<dd>n/a</dd>
-	<dt>Nom technique</dt>
-	<dd><code>gc-featured-link</code></dd>
-</dl>
+	<span class="wb-prettify all-pre hide"></span>
 
-<h2>Objectif</h2>
-<p>Ceci ajoute un lien en vedette court et descriptif vers une page que votre institution doit mettre en évidence. Il doit être utilisé pour attirer l'attention sur les avertissements ou les avis.</p>
+	<dl class="horizontal">
+		<dt>Statut</dt>
+		<dd>Provisiore</dd>
+		<dt>Type</dt>
+		<dd>Conception de Canada.ca</dd>
+		<dt>Dernière revue</dt>
+		<dd>2021-07-02</dd>
+		<dt>Est conforme à</dt>
+		<dd>Spécifications du contenu et de l'architecture de l'information 2.1.2</dd>
+		<dt>Directives</dt>
+		<dd>n/a</dd>
+		<dt>Nom technique</dt>
+		<dd><code>gc-featured-link</code></dd>
+	</dl>
 
-<h2>Exemple pratiques</h2>
+	<h2>Objectif</h2>
+	<p>Ceci ajoute un lien en vedette court et descriptif vers une page que votre institution doit mettre en évidence. Il doit être utilisé pour attirer l'attention sur les avertissements ou les avis.</p>
 
-<h3>Lien en vedette (Provisiore v1.0)</h3>
-<div class="provisional gc-featured-link">
+	<h2>Exemple pratiques</h2>
+
+	<h3>Lien en vedette (Provisiore v1.0)</h3>
+</div>
+
+<section class="provisional gc-featured-link">
```

Comment:
See my comment for the English page, be creative and avoid confusion for the page structure through the heading sequence.